src/Concerns/Bootable.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace BeyondCode\QueryDetector\Concerns;
+
+trait Bootable
```

Comment:
The methods are not used anywhere except for the QueryDetector, so a trait does not make a lot of sense to me.